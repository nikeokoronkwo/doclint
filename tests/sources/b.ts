/**
 * @todo unimplemented
 */
function unimpl() {}

/**
 * @fixme buggy function
 */
function buggy_func() {}
