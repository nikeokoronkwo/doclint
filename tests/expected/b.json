[
    { "id": "doclint/todo", "message": "unimplemented " },
    { "id": "doclint/fixme", "message": "buggy function" }
]